src/styles/calendar.scss

```
@@ -0,0 +1,47 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  padding: 10px;
+  width: $square;
+
+  &__item {
+    display: flex;
+    width: $width;
+    height: $height;
+    border: $border solid black;
+    position: relative;
+    background-color: #eee;
+
```

```suggestion
    background-color: #eee;
    transition-duration: ...
```
Move it here specify for which prop you're applying transition